database/schema/controller.go

```diff
@@ -223,3 +224,35 @@ CREATE TABLE cloud (
     skip_tls_verify     BOOLEAN NOT NULL
 );`[1:]
 }
+
+func externalControllerSchema() string {
+	return `
+CREATE TABLE external_controller (
+    uuid            TEXT PRIMARY KEY,
+    alias           TEXT,
+    ca_cert_uuid    TEXT NOT NULL
+
+    -- TODO (stickupkid): There should be a foreign key constraint on the uuid
+    -- column to a controller(uuid) column. That's not possible because the
+    -- controller table is not created yet.
+);
+
+CREATE TABLE external_controller_address (
+    uuid                        TEXT PRIMARY KEY,
+    address                     TEXT,
+    external_controller_uuid    TEXT NOT NULL,
+    CONSTRAINT          fk_external_controller_address_external_controller
+        FOREIGN KEY         (external_controller_uuid)
+        REFERENCES          external_controller(uuid)
+);
+
+CREATE TABLE external_controller_model (
+    uuid                        TEXT PRIMARY KEY,
```

Comment:
Since a model can only be in one controller, lets just call this `external_model` and have only `uuid` and `controller_uuid`.